### ISSUE TITLE
Injection Press Enchanted Books Not Usable

### ISSUE THREAD

**Issue by EntropyParadigm** [Ipsis/Woot#470]
Using the Injection Press to generate enchanting books cannot be applied to items in an anvil - this might be NBT related:
![image](https://user-images.githubusercontent.com/32775097/100967116-47b1fe80-34e3-11eb-9978-be0f98baa0ec.png)
vs actual Smite V book:
![image](https://user-images.githubusercontent.com/32775097/100967151-60221900-34e3-11eb-975c-0ab20148e745.png)

Playing ATM6 (1.3.3)

**Comment by Ipsis** [comment]
Enchanted books always seem to cause me issues and it is down to the Enchantments vs StoredEnchantments - I though I had it right this time.

There are two ways to apply enchantments one is ItemStack.addEnchantment (Enchantments NBT) and a special one for EnchantedBookItem.addEnchantment (StoredEnchantments NBT). Looks like I'm using the wrong one again .

**Comment by Ipsis** [comment]
Fixed  - output books now have the following NBT style straight from EnchantedBookItem:
Validated by using each of the test produced books in anvil with a diamond sword.

```
{id: "minecraft:enchanted_book", Count: 1b, tag: {StoredEnchantments: [{lvl: 1s, id: "minecraft:power"}]}}
{id: "minecraft:enchanted_book", Count: 1b, tag: {StoredEnchantments: [{lvl: 2s, id: "minecraft:quick_charge"}]}}
{id: "minecraft:enchanted_book", Count: 1b, tag: {StoredEnchantments: [{lvl: 3s, id: "minecraft:loyalty"}]}}
{id: "minecraft:enchanted_book", Count: 1b, tag: {StoredEnchantments: [{lvl: 4s, id: "minecraft:sharpness"}]}}
{id: "minecraft:enchanted_book", Count: 1b, tag: {StoredEnchantments: [{lvl: 5s, id: "minecraft:power"}]}}
```

**Comment by Ipsis** [comment]
Fixed in 1.0.2.1
